packages/twenty-front/src/modules/object-record/record-table/record-table-footer/components/RecordTableColumnAggregateFooterDropdownMoreOptionsContent.tsx

```diff
@@ -0,0 +1,85 @@
+import { getAggregateOperationLabel } from '@/object-record/record-board/record-board-column/utils/getAggregateOperationLabel';
+import { useRecordTableContextOrThrow } from '@/object-record/record-table/contexts/RecordTableContext';
+import { RecordTableColumnAggregateFooterDropdownContext } from '@/object-record/record-table/record-table-footer/components/RecordTableColumnAggregateFooterDropdownContext';
+import { STANDARD_AGGREGATE_OPERATION_OPTIONS } from '@/object-record/record-table/record-table-footer/constants/standardAggregateOperationOptions';
+import { useViewFieldAggregateOperation } from '@/object-record/record-table/record-table-footer/hooks/useViewFieldAggregateOperation';
+import { getAvailableAggregateOperationsForFieldMetadataType } from '@/object-record/record-table/record-table-footer/utils/getAvailableAggregateOperationsForFieldMetadataType';
+import { TableOptionsHotkeyScope } from '@/object-record/record-table/types/TableOptionsHotkeyScope';
+import { DropdownMenuHeader } from '@/ui/layout/dropdown/components/DropdownMenuHeader';
+import { DropdownMenuItemsContainer } from '@/ui/layout/dropdown/components/DropdownMenuItemsContainer';
+import { useDropdown } from '@/ui/layout/dropdown/hooks/useDropdown';
+import { useScopedHotkeys } from '@/ui/utilities/hotkey/hooks/useScopedHotkeys';
+import { useContext, useMemo } from 'react';
+import { Key } from 'ts-key-enum';
+import { IconCheck, IconChevronLeft, MenuItem } from 'twenty-ui';
+
+export const RecordTableColumnAggregateFooterDropdownMoreOptionsContent =
+  () => {
+    const { fieldMetadataId, dropdownId, resetContent } = useContext(
+      RecordTableColumnAggregateFooterDropdownContext,
+    );
+    const { closeDropdown } = useDropdown(dropdownId);
+    const { objectMetadataItem } = useRecordTableContextOrThrow();
+
+    useScopedHotkeys(
+      [Key.Escape],
+      () => {
+        resetContent();
+        closeDropdown();
+      },
+      TableOptionsHotkeyScope.Dropdown,
+    );
+
+    const availableAggregateOperations = useMemo(
+      () =>
+        getAvailableAggregateOperationsForFieldMetadataType({
+          fieldMetadataType: objectMetadataItem.fields.find(
```

Comment:
What happens if field is undefined?

Reply:
Oh ok, looks like if the field can't be retrieved we pass undefined to getAvailableAggregateOperationsForFieldMetadataType which simply returns "Count" as default aggregate 👍